src/Microsoft.Extensions.Logging.Console/Properties/AssemblyInfo.cs

```diff
@@ -3,6 +3,8 @@
 
 using System.Reflection;
 using System.Resources;
+using System.Runtime.CompilerServices;
 
 [assembly: AssemblyMetadata("Serviceable", "True")]
-[assembly: NeutralResourcesLanguage("en-us")]
\ No newline at end of file
+[assembly: NeutralResourcesLanguage("en-us")]
+[assembly: InternalsVisibleTo("Microsoft.Extensions.Logging.Test")]
```

Comment:
IVT is only from a product assembly to its own unit test assembly. Never use across misc assemblies.